src/exp.jl

```diff
@@ -131,7 +180,7 @@ end
     mul!(temp2, Q1, transpose(Qim))
     mul!(Q1, Q2, transpose(Qr))
     Q1 .-= temp2
-    return Q1
+    return skewhermitian!(Q1)
```

Comment:
This doesn't seem correct — isn't the `cos` of a skew-hermitian matrix Hermitian, not skew-hermitian?

Reply:
Nevermind, this is in sin, not cos.